packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -1178,6 +1183,21 @@
                     "required": true
                 }
             ]
+        },
+        {
+            "name": "trace_event",
```

Comment:
Is there a reason not to use a conventional name such as `toolkit_trace` ? The prefix is useful as a "category". If we introduce new prefixes, there should be a reason.